automation/Dockerfile

```diff
@@ -1,16 +1,21 @@
 FROM nmstate/centos7-nmstate-base
 
 RUN yum -y install \
-        python2-pip \
-        rpm-build \
-        git \
-        python2-devel \
         dnsmasq \
+        git \
         iproute \
+        python2-devel \
+        python2-pip \
+        python36-pip \
         radvd \
-    && \
-    pip install -U pip pytest==4.2.1 pytest-cov==2.6.1 python-coveralls && \
-    yum clean all
+        rpm-build \
+    && yum clean all
+
+RUN    pip install --upgrade pip pytest==4.2.1 pytest-cov==2.6.1
+
+RUN    pip3 install --user python-coveralls
```

Comment:
Can you explain why old `pip install python-coveralls` does not works?

Reply:
I added the reasons for using Python3 to the commit message already. Is it enough?